src/Makefile.am

```diff
@@ -476,6 +476,7 @@ CLEANFILES += univalue/*.gcda univalue/*.gcno
 CLEANFILES += wallet/*.gcda wallet/*.gcno
 CLEANFILES += wallet/test/*.gcda wallet/test/*.gcno
 CLEANFILES += zmq/*.gcda zmq/*.gcno
+CLEANFILES += obj/build.h
 
 DISTCLEANFILES = obj/build.h
```

Comment:
It can probably be removed from DISTCLEANFILES in that case?
Not sure what the rules here are @theuni

Reply:
Yep, agreed.

IIRC the intention here was for build.h to be generated when bootstrapping, in which case ```make clean``` should not clean it. But we don't currently do that and all that needs to be re-worked anyway.

Reply:
Fixed! :-)